containerd-shim-v2/create.go

```diff
@@ -136,23 +137,6 @@ func loadSpec(r *taskAPI.CreateTaskRequest, netns string) (*oci.CompatOCISpec, s
 		return nil, "", err
 	}
 
-	//set the network namespace path
-	//this set will be applied to sandbox's
-	//network config and has nothing to
-	//do with containers in the sandbox since
-	//networkNamespace has been ignored by
-	//kata-agent in sandbox.
-
-	for _, n := range ociSpec.Linux.Namespaces {
-		if n.Type != specs.NetworkNamespace {
-			continue
-		}
-
-		if n.Path == "" {
-			n.Path = netns
```

Comment:
Just say，this code is wrong, n is a temporary value， so n.Path is not changed

Reply:
Good catch @Ace-Tang !!